js/mail.js

```diff
@@ -671,6 +678,14 @@ var Mail = {
 					});
 					// Remove spinner when loading finished
 					$('iframe').parent().removeClass('icon-loading');
+					// Add body content to inline reply
+					var text = $(this).contents().find('body')[0].innerText;
+					document.getElementsByClassName('reply-message-body')[0].innerText = (
+						'\n\n\n' +
+						data.from + ' – ' +
+						moment(data.dateIso).format('ddd, DD. MMM YYYY HH:mm') + '\n> ' +
```

Comment:
I suppose the comment I made about the date format would be solved here by using `ddd, D. MMM YYYY H:mm` instead?